partner_country_lang/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022-2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    lang = fields.Selection(compute="_compute_lang")
```

Comment:
Always add a comment indicating that this is an override

```suggestion
    # Add the compute method on the existing field
    lang = fields.Selection(compute="_compute_lang")
```